dvc/remote/oss.py

```diff
@@ -107,18 +107,22 @@ def list_cache_paths(self):
     def _upload(
         self, from_file, to_info, name=None, no_progress_bar=False, **_kwargs
     ):
-        cb = None if no_progress_bar else Callback(name)
-        self.oss_service.put_object_from_file(
-            to_info.path, from_file, progress_callback=cb
-        )
+        with Tqdm(
+            desc_truncate=name, disable=no_progress_bar, bytes=True
```

Comment:
I presume `oss2.Bucket.put_object_from_file:progress_callback` uses `bytes` as a unit (documentation is in Mandarin and the English translation is unhelpful)